Pellicola/Classes/PellicolaPresenter.swift

```diff
@@ -137,9 +139,15 @@ public final class PellicolaPresenter: NSObject {
         
         let leftBarButtonType: AssetCollectionsViewController.LeftBarButtonType = isRootLevel ? .dismiss : .back
         let assetCollectionsVC = AssetCollectionsViewController(viewModel: viewModel, style: style, leftBarButtonType: leftBarButtonType)
-        assetCollectionsVC.didSelectImages = { [weak self] images in
-            self?.dismissWithImages(images)
+        assetCollectionsVC.didStartProcessingImages = { [weak self] in
+            self?.didStartProcessingImages?()
+        }
+        
+        assetCollectionsVC.didFinishProcessingImages = { [weak self] urls in
+            self?.didFinishProcessingImages?(urls)
+            self?.navigationController.dismiss(animated: true)
```

Comment:
would it make sense to call didFinishProcessingImages in the navigationcontroller dismiss completion?

Reply:
Ok